### PR TITLE
Updated preview entries

### PR DESCRIPTION
**Pull request by oscargus**
While working on #1602 I realized that the preview entries used deprecated fields, so I updated them and improved the casing a bit.

### CODE REVIEW

**Comment by simonharrer** [comment]
I would prefer a single class which contains a preview entry, which is then used for both style and preview.  What do you think?

**Comment by oscargus** [comment]
Shouldn't be a problem and makes sense. I guess there are two for
historical reasons (plugin). I'm even wondering if there isn't a third one
although it is somehow hidden in the code and never seen... (Only used as
some sort of initialization.) Didn't find it yesterday though.

**Comment by oscargus** [comment]
Now there is a single preview entry.

**Comment by simonharrer** [comment]
I would always create a new bib entry instead of reusing it. Returning a copy is more robust, as you never know who modifies this entry, resulting in strange preview behavior. With this in place, just go ahead and merge.

File: src/main/java/net/sf/jabref/logic/util/TestEntry.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package net.sf.jabref.logic.util;
```
Comment (tobiasdiez) [inline src/main/java/net/sf/jabref/logic/util/TestEntry.java]: Not sure that it should life in logic.

Comment (oscargus) [inline src/main/java/net/sf/jabref/logic/util/TestEntry.java]: I'm open for suggestions on location. No dependency issues either way.

Comment (simonharrer) [inline src/main/java/net/sf/jabref/logic/util/TestEntry.java]: I merged it. We can discuss moving the class somewhere else somewhere else. ;)

**Pull request merged**